contracts/bridge/BridgeOperator.sol

```diff
@@ -0,0 +1,74 @@
+pragma solidity ^0.4.24;
+
+import "../externals/openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+import "../externals/openzeppelin-solidity/contracts/token/ERC20/ERC20Mintable.sol";
+import "../externals/openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "../externals/openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+contract BridgeOperator is Ownable {
+    mapping(address => bool) public operators;
+    mapping(bytes32 => mapping(address => bool)) public signedTxs; // <sha3(type, args, nonce), <singer, vote>>
+    mapping(bytes32 => uint64) public signedTxsCounts; // <sha3(type, args, nonce)>
+    mapping(bytes32 => bool) public committedTxs; // <sha3(type, nonce)>
+    mapping(uint64 => uint64) public operatorThresholds; // <tx type>
+    mapping(uint64 => uint64) public configurationNonces; // <tx type, nonce>
+
+    enum VoteType {
+        ValueTransfer,
+        Configuration,
+        Max
+    }
+
+    constructor() internal {
+        for (uint64 i = 0; i < uint64(VoteType.Max); i++) {
+            operatorThresholds[uint64(i)] = 1;
+        }
+    }
+
+    modifier onlyOperators()
+    {
+        require(operators[msg.sender]);
+        _;
+    }
+
+    // onlySequentialNonce checks sequential nonce increase.
+    function onlySequentialNonce(VoteType _voteType, uint64 _requestNonce) internal view {
```

Comment:
Will you use this function with a `VoteType.ValueTransfer` parameter value? 
Currently, this function only gets `VoteType.Configuration` as the first parameter. 
If the function is not going to get `VoteType.ValueTransfer`, it can remove the first parameter type.
(Even it the function can be divided into two functions for each `VoteType`.)
Then, it can be used as a modifier.